build.gradle

```diff
@@ -29,20 +29,22 @@ buildscript {
 }
 
 plugins {
-  id 'com.github.ben-manes.versions' version '0.53.0'
   id 'idea'
   id 'jacoco'
   id 'java-library'
+  id 'com.diffplug.spotless' version '8.1.0'
+  id 'com.github.ben-manes.versions' version '0.53.0'
+  id 'com.github.spotbugs' version '6.4.4' apply false
+  id 'com.gradleup.shadow' version '8.3.9' apply false
+  id 'io.swagger.core.v3.swagger-gradle-plugin' version "${swaggerVersion}"
+  id 'org.nosphere.apache.rat' version '0.8.1'
+  id 'org.openrewrite.rewrite' version '7.22.0' apply false
```

Comment:
just wanted to align and give natural order.

Reply:
correct. This tool yet another SCA already running in [checkstyle](https://github.com/checkstyle/checkstyle/blob/e2c1b6a8cd1ea056a4ec275e319442ef6617014f/rewrite.yml#L20) and [spotless](https://github.com/diffplug/spotless/pull/2756). 

Highly welcome any feedback, thx.